pages/contribute.tsx

```diff
@@ -19,12 +31,123 @@ function Contribute(): JSX.Element {
             site_name: 'open source at ACM at UCLA',
           }}
         />
-        <h1>
-          get started
-        </h1>
+        <div className='row spaced-row'>
+          <h1>get started </h1>
+          <a href='#good-first-issues'><button>good first issues</button></a>
+        </div>
+        <hr />
+        <h2>
+          what is open source?
+        </h2>
+        <p>
+          Open source software is code that is made freely available for <b>anyone</b> to
+          use or enhance! Here at ACM, we take pride in <b>all</b> of our projects being open-source
+          and open for anyone to work on.
+        </p>
+        <h2>
+          what makes open source possible?
+        </h2>
+        <p>
+          The main back bone of open source initiatives are tools that can make collaboration and sharing of code as
+          simple as possible. Git and <ELink link='https://github.com/'>Github</ELink> are two really popular tools
+          that do just that used by companies like Google, Meta, and more, and is what we use at ACM as well.
+        </p>
+        <h2>
+          how can i contribute?
+        </h2>
+        <p>
+          We&apos;ll quickly go over how we can use git/github to work on some open source projects, and if
+          you want to learn more about the specifics of what we&apos;ll cover, you can check out {' '}
+          <ELink link='https://git-scm.com/doc'>git&apos;s documentation</ELink>
+          {' '} or our writeups on {' '}
+          <ELink link='https://github.com/uclaacm/centralized-intern-training/tree/main/01_html_css_github#git-vs-github'>
+            git workflows
+          </ELink>
+          {' '} and {' '}
+          <ELink link='https://github.com/uclaacm/centralized-intern-training/tree/main/02_intermediate_css_github_workflows#github-as-a-collaboration-tool'>
+            github as a collaboration tool
+          </ELink>
+          {' '}
+          to get a closer look at what we&apos;re doing!
+        </p>
+        <h3>
+          jumping into contributing on a project
+        </h3>
+        <p>
+          There are thousands of open-source projects out there that it can get hard to even find where to start!
+          Luckily, some <strong>repositories</strong>, or the main hub of projects, have <strong>issues</strong>, or
+          project tasks, marked as <strong>good first issue</strong>.
+          These are great starting points for people to hop in and contribute!
+        </p>
+
+        <h2 id='good-first-issues'>
+          acm&apos;s projects w/ good first issues
+        </h2>
+        <div className='card'>
+          <div className='card-body'>
+            {displayedgfiProjects}
+          </div>
+        </div>
```

Comment:
This is a pretty large content box; I would suggest either making it collapsible, only showing the first 3, or linking to some other page. If I saw this, I wouldn't know to scroll down and read the other FAQs!

Reply:
I think putting it at the end of the page and linking to it or making it its separate page would work, would love to hear your thoughts!

Reply:
Not a huge fan of end of page now looking at it. I think separate page makes the most sense, and add it to the navbar/the front four cards (since some of those links are meaningless right now).

Reply:
Hm, we'll definitely have to figure out how to sort out the navbar if we'll have the `ucla-opensource` page up there as well, it might get a bit crowded. I'll probably add a link to it on the front-page for now.